docs/reference/schemas/packagings/packaging_component.yaml

```diff
@@ -0,0 +1,41 @@
+title: Packaging component
+x-stoplight:
+  id: 5jrtlek3arbg2
+type: object
+description: |-
+  Each packaging component has different properties to specify how many there are, its shape, material etc.
+
+  The shape, material and recycling properties are mapped to one entry in the packaging_shapes, packaging_materials and packaging_recycling taxonomies, and the value of the property is the canonical name of the taxonomy entry (e.g. en:bottle).
+
+  They may contain values that could not yet get matched to their respective taxonomy, in which case they will contain a free text value prefixed with the language code of this text value (e.g. "fr:Bouteille sphérique" might have been entered by a French user to indicate it is a spherical bottle).
+properties:
+  number:
+    type: integer
+    description: Number of units of this packaging component contained in the product (e.g. 6 for a pack of 6 bottles)
+  shape:
+    type: string
+    description: 'Shape, canonicalized with the packaging_shapes taxonomy.'
```

Comment:
when used as the Post body, shouldn't we accept plain values that we will match against the taxonomy ?

Also I imagine sometime something is not yet part of the taxonomy, so it's just a normalized string, right ?
Should we precise if it's part of the taxonomy with a specific field ? (for display only)

Reply:
Right, as input, we accept anything (either plain text in a specified language, or a taxonomy entry).
As output, we return the canonical taxonomy entry, or plain text prefixed by the language code if we couldn't match it to a taxonomy.

I was hoping we could avoid duplicating the packaging_component.yaml (one for input and one for output)

Reply:
It's ok if you just precise it in the description.